### DISCUSSION TITLE
Position specific stimulus

### DISCUSSION THREAD

**Discussion by Sigmalalala** [bonsai-rx/bonsai#1171] [Q&A]
Hello, I'm a new Bonsai user and i want to give a stimulus when the mouse reaches a specific area from the camera. But the problem I'm having is that i can't combine the condition of the x-axis with the condition of the y-axis. I have also used "Sample" to combine them, but it still didn't work. Is there any way I can get this to work? Thank you so much!
![图片1](https://user-images.githubusercontent.com/118099060/205483295-8b669419-6ea7-4e8f-8f02-5d9a8d222ebf.png)

**Comment by bruno-f-cruz** [comment]
Hey,
You need to combine the two booleans on a single one. Look for the `BitWise` or `Logical` operators in the toolbox. They should be added after your `Zip`.

**Reply by Sigmalalala** [reply]
Wow, that's magic! You helped me to solve this problem perfectly. Thank you so much! It's really nice of you!
